guides/common/modules/proc_enabling-puppet.adoc

```diff
@@ -0,0 +1,27 @@
+[id="enabling-puppet_{context}"]
+= Enabling Puppet Integration with {Project}
+
+The installation of {Project} does not have any Puppet integration configured by default.
```

Comment:
I'm unsure about configured vs enabled. I'm leaning to enabled since by default the code isn't even present.

Reply:
Likewise.